README.md

```diff
@@ -56,8 +56,53 @@ The above workflow (when manually triggered) will
 ![image](https://user-images.githubusercontent.com/2510597/140506212-4938e44d-0662-4dc5-9fb1-c3f59fe075a6.png)
 
 - When you merge this PR, it will request an Npm OTP from Optic. (If you close the PR, nothing will happen)
-- Upon successful retrieval of the OTP, it will publish the package to Npm.
+- You need to define Npm and Optic tokens in the Github secrets for each user that will receive the otp
```

Comment:
note that this is not mandatory, because if you don't specify those inputs simply there won't be any publish to npm, which is what we want because this action can be used to release other things (e.g. applications or github actions)